test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -3,13 +3,33 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
+import java.awt.*;
```

Comment:
This is a coding standard violation. Please see how to avoid wildcard violations - posted in the slack channel.